functional-tests/common/com.liferay.ide.functional.swtbot/src/com/liferay/ide/functional/swtbot/eclipse/page/PreferencesDialog.java

```diff
@@ -29,6 +30,10 @@ public PreferencesDialog(SWTBot bot) {
 		super(bot, PREFERENCES, CANCEL, APPLY_AND_CLOSE);
 	}
 
+	public Button applyAndCloseBtn() {
```

Comment:
could you help check if we already have a method to execute apply and close function? If found, then we can delete this.